package.json

```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "axios": "^0.24.0",
+    "eslint": "^8.57.1",
```

Comment:
Why was it moved from being a dev dependency to here?

Reply:
To resolve the error, I had to uninstall and then reinstall the package, which added it to the dependencies instead of the dev dependencies.

Reply:
Can we try to reinstall as dev dependency?